packages/react-native-web/src/modules/usePressEvents/PressResponder.js

```diff
@@ -345,6 +347,7 @@ export default class PressResponder {
         if (!disabled && isValidKeyPress(event)) {
           if (this._touchState === NOT_RESPONDER) {
             start(event, false);
+            this._responderElement = target;
```

Comment:
Can we set this back to `null` when the interaction is over

Reply:
Done 👍